extension/src/project/projectTree.ts

```diff
@@ -236,5 +236,6 @@ export function addLispFileNode2ProjectTree(root: ProjectNode, fileName: string,
     fileNode.fileExists = fs.existsSync(fileName);
     fileNode.rawFilePath = rawFilePath;
     fileNode.document = ReadonlyDocument.open(fileName);
-    root.sourceFiles.push(fileNode);
+    fileNode.document.updateAtomsForest();
+    root.sourceFiles.push(fileNode);    
```

Comment:
Added a pre-load on the PRJ atoms forest since it is an explicit act to load the project and makes sense for them to suffer a load time vs the entire workspace. From what you were saying, I can expect these to potentially be very large.